crates/bevy_window/src/lib.rs

```diff
@@ -39,8 +39,8 @@ impl Default for WindowPlugin {
 /// A [`Plugin`] that defines an interface for windowing support in Bevy.
 pub struct WindowPlugin {
     /// Settings for the primary window. This will be spawned by
-    /// default, if you want to run without a primary window you should
-    /// set this to `None`.
+    /// default, bundled with the marker component [`PrimaryWindow`](PrimaryWindow).
```

Comment:
```suggestion
    /// default, with the marker component [`PrimaryWindow`](PrimaryWindow).
```
Not sure the word "bundled" add enough value here to be used as bundles don't really have an existence and it's not even added as a single bundle.